src/themes/elife/styles.css

```diff
@@ -92,3 +95,12 @@ main {
     }
   }
 }
+
+:--datePublished {
+  font-family: 'Noto Sans', Arial, Helvetica, sans-serif;
+  font-size: calc(var(--text-size-rem-small) * 1rem);
+  line-height: calc(1.5rem / var(--text-size-rem-small));
```

Comment:
Should probably have the `1.5rem` in a custom property too.

Reply:
@davidcmoulton What should we call this then? Are you sure this is not fragmenting this too much?

Reply:
Cannot think of a suitable name for this if we are going to use it as a global. I have set it up as if we were going to use it for line-height.